satellite/gracefulexit/getnodes_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/overlay"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func TestGetExitingNodes(t *testing.T) {
```

Comment:
Should we also have a test for the `Update` method?

Reply:
since this test calls `UpdateExitStatus` I wasn't thinking there's a need for a specific test. do you think there are different cases that need to be tested?

Reply:
Test updated, but per Nat's comment above, there is not a separate test for `Update`